### PR TITLE
Feature Works and contact

### PR DESCRIPTION
**Pull request by elfin-git**
I updated the project with a works section and a contact section

### CODE REVIEW

**Comment by elfin-git** [comment]
For some reason, my last section has overlapping elements. Can you please help me identify the error I made?

File: index.html [hunk 1]
```diff
@@ -64,5 +66,121 @@ <h3>Software Developer</h3>
...
+         <li><span class="div"></span></li>
+         <li class="tag">Bootsrap</li>
+         <li><span class="div"></span></li>
+         <li class="tag">Ruby</li>
+       </ul>
+       <button class="btn" type="button" onclick="">See project</button>
+    </div>
+  </div>
+  <div class="mps-grid">
+    <img class="story-img" alt="image story" src="image_processing20200716-21735-f07s4k.png">
```
Comment (ishimwezachee) [inline index.html:117]: - [OPTIONAL ] You should  fix your images, for you project to be more professional, you need to fix the issue of image loading in your website 
image

File: index.html [hunk 2]
```diff
@@ -64,5 +66,121 @@ <h3>Software Developer</h3>
...
+          <li class="tag">html</li>
+          <li><span class="div"></span></li>
+          <li class="tag">Bootsrap</li>
+          <li><span class="div"></span></li>
+          <li class="tag">Ruby</li>
+       </ul>
+       <button class="btn" type="button" onclick="">See project</button>
+     </div>
+   </div>
+  </section>
```
[the remark this answers is not in the document]
Comment (elfin-git) [reply index.html:157]: @ishimwezachee do you mean to say that the languages, frameworks &  skills are the about me? The template I have chosen is quite different from the rest, so I am not too clear on what parts should be sections...

**Comment by elfin-git** [comment]
I have made a section from the header to the social media icons. This is the about me section, as my template is different from the rest.

**Review by Ismailco — APPROVED** [review]
### Hi @elfin-git,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**